web/packages/teleport/src/config.ts

```diff
@@ -456,6 +456,9 @@ const cfg = {
     botInstancePath:
       '/v1/webapi/sites/:clusterId/machine-id/bot/:botName/bot-instance/:instanceId',
     botInstancesPath: '/v1/webapi/sites/:clusterId/machine-id/bot-instance',
+    bot: {
+      update: '/v2/webapi/sites/:clusterId/machine-id/bot/:botName',
+    },
```

Comment:
you can do this in another PR, but i would also put the `v1` update here too, so then you can mark this for deletion, and when it comes to deleting it, any place that references this can be easily spotted with linter

```suggestion
    bot: {
      // TODO .... mark for deletion....
      update: '/v1/webapi/sites/:clusterId/machine-id/bot/:botName',
      updateV2: '/v2/webapi/sites/:clusterId/machine-id/bot/:botName',
      // i would also add in the rest like "list", "get", "delete"
      // and remove the `botsPath`
    },
```